cmake/templates/LeptonicaConfig.cmake.in

```diff
@@ -22,6 +22,11 @@
 
 include(${CMAKE_CURRENT_LIST_DIR}/LeptonicaTargets.cmake)
 
+find_package(JPEG CONFIG)
```

Comment:
My thoughts were that the REQUIRED is already used in the CMakeLists so this find_package will succeed regardless.